TTKDTrainingApp/src/redux/selectors.js

```diff
@@ -26,17 +27,14 @@ export const getCategoryContent = (store, categoryId) =>
 export const getContentFromId = (store, contentId) =>
   getAllContent(store).find(content => content.id === contentId);
 
-export const getContentOwnVideoUri = (store, contentId = -1) => {
-  const content = getContentFromId(store, contentId);
-  return content && content.video_path;
-};
-
-export const getContentVideoUri = (store, contentId) =>
-  getAllVideoUris(store)[contentId];
-
 export const getContentOwnCachedVideoPath = (store, contentId) =>
   getAllCachedVideoPaths(store)[contentId];
 
+export const getContentImageUri = (store, contentId) => {
+  const {title, thumbnail_path} = getContentFromId(store, contentId);
```

Comment:
how do you get the thumbnail ?

Reply:
The thumbnail itself is stored in S3, so this is the method to generate the URL for that file. In the future we'll most likely remove the requirement of `title` and simply have all the info we need in `thumbnail_path`